frontend/src/lib/types/global.d.ts

```diff
@@ -85,6 +85,29 @@ declare global {
     politicalExperience: string;
   }
 
+  /** These are all the DaisyUI colors supported by the application. */
+  type Color =
```

Comment:
Having some types for the colours may be a good idea, but it may be hard to suggest the right prefix for the colours. For example the `bg-{COLOR}`, `text-{COLOR}`and so on

Reply:
I think this type is still usable, because it can be used in component props. Those should probably anyway accept just the color without `fill` or `text` prefixed to it.

Reply:
If we use this only for the components, could you add the types inside the folder `components`. if not maybe a comment saying that these colors are used in the components.

Reply:
I moved the type definition and the `FILLS` record to `$lib/components/shared/colors/`.